### PR TITLE
[T2A2][T13-A1]Gabriel Gheorghian

### PR DESCRIPTION
**Pull request by GabrielGhe**

### CODE REVIEW

**Comment by GabrielGhe** [comment]
ready for review

**Comment by mauris** [comment]
@GabrielGhe do you have a team number yet?

**Comment by GabrielGhe** [comment]
@mauris Yes

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -254,20 +267,20 @@ private static void echoUserCommand(String userCommand) {
...
-
-        if (args.length == 1) {
-            setupGivenFileForStorage(args[0]);
-        }
-
-        if(args.length == 0) {
-            setupDefaultFileForStorage();
-        }
-    }
+//    private static void processProgramArgs(String[] args) {
```
Comment (okkhoy) [inline src/seedu/addressbook/AddressBook.java]: remove redundant code. don't retain commentedversions

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -203,50 +203,59 @@ public static void main(String[] args) {
         
         // Begin process args
         if (args.length >= 2) {
```
Comment (okkhoy) [inline src/seedu/addressbook/AddressBook.java:207]: yes! its quite ugly, but no need paper bag yet :-P

File: src/seedu/addressbook/AddressBook.java [hunk 3]
```diff
@@ -606,11 +608,26 @@ private static String executeClearAddressBook() {
      *
      * @return feedback display message for the operation result
      */
-    private static String executeListAllPersonsInAddressBook() {
+    private static String executeListAllPersonsInAddressBook(String commandArgs) {
```
Comment (okkhoy) [inline src/seedu/addressbook/AddressBook.java:611]: if it is a single argument, name should be singular form - by coding standards

**Comment by okkhoy** [comment]
@GabrielGhe for this tutorial it is OK to have all in one PR; going forward, please submit one PR per activity.
- in general good work overall.
  code improvements specific:
- in general, pay more attention to coding standards violation.
- some useful ones include checking how conditional statements are written, naming conventions etc.,

**Pull request closed without merging**